src/ctia/stores/es/mapping.clj

```diff
@@ -635,6 +635,16 @@
       :tokenizer "keyword"
       :type "custom"}}}})
 
+(def investigation-mapping
```

Comment:
I ask that reviewers pay special attention to this mapping.  The goals is that we can add anything to an Investigation so that we can postpone defining a schema.  I think this should work, but I'm not 100% sure.

Reply:
perhaps you just want to store and not make dynamic data available for search.
you want to make common entity fields like `:description` searchable.
In this case I suggest putting all that data on a top level field and disable indexing for it:

https://www.elastic.co/guide/en/elasticsearch/reference/current/enabled.html

Reply:
This is a good suggestion.  I talked it over with Craig, and we decided to just set `:dynamic = false`.  The [doc](https://www.elastic.co/guide/en/elasticsearch/reference/current/dynamic.html) says that, "Newly detected fields are ignored. These fields will not be indexed so will not be searchable but will still appear in the _source field of returned hits."  This seems like a reasonable compromise.